src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/dto/DataDto.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+public class DataDto {
+    @JsonInclude(NON_NULL)
+    @JsonProperty("partnership_name")
+    private String partnershipName;
+
+    @JsonInclude(NON_NULL)
+    @JsonProperty("name_ending")
+    private String nameEnding;
```

Comment:
Rather than have a datatype of String here, I thought we'd define a Java enum with the allowed values, matching those defined in the Node SDK enum.

Reply:
I was in two minds about this but given the SDK has a enum then the API should follow suit. Have added a enum and some mapping so that the actual value can be mapped to and mapped from DAO.